apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -339,22 +351,86 @@
                                     </div>
                                     <br>
                                 <% } else { %>
-                                    <div class="external-login blurring external-login-dimmer">
-                                        <div class="field">
-                                            <button
-                                                class="ui button fluid"
-                                                onclick="handleNoDomain(this,
-                                                    '<%=Encode.forJavaScriptAttribute(Encode.forUriComponent(idpName))%>',
-                                                    '<%=Encode.forJavaScriptAttribute(Encode.forUriComponent(idpEntry.getValue()))%>')"
-                                                id="icon-<%=iconId%>"
-                                                title="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%> <%=Encode.forHtmlAttribute(idpName)%>"
-                                            >
-                                                <img class="ui image" src="<%=Encode.forHtmlAttribute(imageURL)%>">
-                                                <span><%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%> <%=Encode.forHtmlContent(idpName)%></span>
-                                            </button>
+                                    <% if (StringUtils.equals(idpEntry.getValue(),GOOGLE_AUTHENTICATOR)) {
+                                            isGoogleIdp = true;
+                                            IdentityProviderDataRetrievalClient identityProviderDataRetrievalClient = new IdentityProviderDataRetrievalClient();
+                                            List<String> configKeys = new ArrayList<>();
+                                            configKeys.add("ClientId");
+                                            configKeys.add("callbackUrl");
+                                            configKeys.add("IsGoogleOneTapEnabled");
+
+                                            try {
+                                                Map<String,String> idpConfigMap = identityProviderDataRetrievalClient.getFederatedIdpConfigs(tenantDomain, GOOGLE_AUTHENTICATOR, idpName, configKeys);
+                                                if (idpConfigMap != null && !idpConfigMap.isEmpty()) {
+                                                    GOOGLE_CLIENT_ID = idpConfigMap.get("ClientId");
+                                                    GOOGLE_CALLBACK_URL = idpConfigMap.get("callbackUrl");
+                                                    String oneTapEnabledString = idpConfigMap.get("IsGoogleOneTapEnabled");
+                                                    if (StringUtils.isNotEmpty(oneTapEnabledString)) {
+                                                        GOOGLE_ONE_TAP_ENABLED = oneTapEnabledString.equals("true");
```

Comment:
Can't we use `Boolean.parse()` ?

Reply:
Fixed 9b4b2bc25c47eef914238e8c4592fd165da371c6